docs/how-to/profiling-python-scripts.rst

```diff
@@ -0,0 +1,334 @@
+.. meta::
+   :description: Omnitrace documentation and reference
+   :keywords: Omnitrace, ROCm, profiler, tracking, visualization, tool, Instinct, accelerator, AMD
+
+****************************************************
+Profiling Python scripts
+****************************************************
+
+`Omnitrace <https://github.com/ROCm/omnitrace>`_ supports profiling Python code at the 
+source level and the script level.
+Python support is enabled via the ``OMNITRACE_USE_PYTHON`` and the 
+``OMNITRACE_PYTHON_VERSIONS="<MAJOR>.<MINOR>`` CMake options.
+Alternatively, to build multiple Python versions, use 
+``OMNITRACE_PYTHON_VERSIONS="<MAJOR>.<MINOR>;[<MAJOR>.<MINOR>]"``,
+and ``OMNITRACE_PYTHON_ROOT_DIRS="/path/to/version;[/path/to/version]"`` instead of ``OMNITRACE_PYTHON_VERSION``.
+When building multiple Python versions, the length of the ``OMNITRACE_PYTHON_VERSIONS`` 
+and ``OMNITRACE_PYTHON_ROOT_DIRS`` lists must
+be the same size.
+
+.. note::
+
+   When using Omnitrace for Python, the Python interpreter major and minor version (e.g. 3.7) 
```

Comment:
is this product or version name?
 Omnitrace for Python
 
 If not, perhaps something like:
 When using OT with Python, ...

Reply:
No, it's not a specific product, Omnitrace can also be used to profile Python programs. I see how this might be confusing, so I'll change it to "When using Omnitrace with Python programs" ...